### PR TITLE
Fix: Kit/Manager - Prevent default kit from being reset when site cache. (ED-2396) 

### PR DESCRIPTION
**Pull request by nisan250**

### CODE REVIEW

**Review by StyleShit — APPROVED** [review]
LGTM

File: core/kits/manager.php [hunk 1]
```diff
@@ -23,13 +23,13 @@ class Manager {
 	const E_HASH_COMMAND_OPEN_SITE_SETTINGS = 'e:run:panel/global/open';
 
 	public function get_active_id() {
-		$id = get_option( self::OPTION_ACTIVE );
+		$id = get_option( self::OPTION_ACTIVE, null );
```
Comment (matipojo) [inline core/kits/manager.php:26]: Why?

**Review by matipojo — CHANGES_REQUESTED** [review]

File: core/upgrade/upgrades.php [hunk 2]
```diff
@@ -646,6 +646,16 @@ public static function _v_2_8_3_recalc_usage_data( $updater ) {
 		return self::recalc_usage_data( $updater );
 	}
 
+	public static function _v_3_3_0_kit_options_update_autoload() {
+		global $wpdb;
+		$id = get_option( 'elementor_active_kit', null );
```
Comment (matipojo) [inline core/upgrade/upgrades.php:651]:
```suggestion
		$id = get_option( 'elementor_active_kit' );
```

File: core/upgrade/upgrades.php [hunk 3]
```diff
@@ -646,6 +646,16 @@ public static function _v_2_8_3_recalc_usage_data( $updater ) {
...
+
+		if ( ! $id ) {
```
Comment (matipojo) [inline core/upgrade/upgrades.php:653]:
```suggestion
		if ( $id ) {
```

File: core/upgrade/upgrades.php [hunk 4]
```diff
@@ -646,6 +646,16 @@ public static function _v_2_8_3_recalc_usage_data( $updater ) {
...
+			// Query the row and update autoload
+			$wpdb->query( $wpdb->prepare( "UPDATE $wpdb->options SET autoload='no' WHERE option_name=%d", 'elementor_active_kit' ) );
```
Comment (matipojo) [inline core/upgrade/upgrades.php:655]:
```suggestion
			$wpdb->query( $wpdb->prepare( "UPDATE $wpdb->options SET autoload='no' WHERE option_name='elementor_active_kit'" ) );
```

File: core/upgrade/upgrades.php [hunk 5]
```diff
@@ -646,6 +646,16 @@ public static function _v_2_8_3_recalc_usage_data( $updater ) {
...
+		}
+	}
+
```
Comment (matipojo) [inline core/upgrade/upgrades.php:658]: Add a test for it.

**Review by matipojo — CHANGES_REQUESTED** [review]

**Comment by matipojo** [comment]
According to @KingYes it should be solved in another approach

**Pull request closed without merging**